__tests__/test.js

```diff
@@ -166,15 +232,23 @@ describe('createIdObject', () => {
     });
 });
 
-describe('filterContributorByTime', () => {
-    sinon.spy(console, 'log');
-
-    it('must show the contributor email', () => {
+describe('didTheyQualify', () => {
+    it('returns true if two contributions are within the time range and the minimum number of contributions is 2', () => {
         const idObject = createIdObject(['mockedUser', 'mockedUser@user.com'], mockedEvents);
         const moments = parseDatesFromArgv();
-
-        filterContributorByTime(idObject, moments);
-
-        expect(console.log.calledWith('mockedUser@user.com')).toEqual(true);
+        expect(didTheyQualify(idObject, moments)).toBeTruthy();
+    });
+    it('returns false if there is only 1 valid contribution, minimum number of contributions is higher than 1', () => {
+        const idObject = createIdObject(['mockedUser', 'mockedUser@user.com'], mockedEvents[0]);
```

Comment:
It feels like this should pass [mockedEvents[0]] (an array), since the other call to createIdObject took an array in that position.

Reply:
Agreed! Thanks.